types/block.go

```diff
@@ -587,9 +587,9 @@ const (
 const (
 	// Max size of commit without any commitSigs -> 82 for BlockID, 8 for Height, 4 for Round.
 	MaxCommitOverheadBytes int64 = 94
-	// Commit sig size is made up of 64 bytes for the signature, 20 bytes for the address,
+	// Commit sig size is made up of 96 bytes for the signature, 20 bytes for the address,
 	// 1 byte for the flag and 14 bytes for the timestamp.
-	MaxCommitSigBytes int64 = 109
+	MaxCommitSigBytes int64 = 131 + 10 // where's the 10 from?
```

Comment:
Can you open an issue whose goal is to find out where they come from?

Reply:
https://github.com/cometbft/cometbft/issues/2838